package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "explicit",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
Reducing the effort is very considerate ❤️ but please don't change the version in a PR. It should be changed just before a release.

Reply:
You're absolutely right, force of habit. Will change.